### ISSUE TITLE
Multiple Handle-It Work Items per application and Project

### ISSUE THREAD

**Issue by dandahle** [dandahle/Catalyst-AppVetting#226]
It seems that only one Handle-It Work Item shows up in a Handle-It project in the Project menu. At times there are several (easy) things for a Handle-It team to do when they visit a client, and it would be nice to be able to enter each Work Items separately so the Handle-It team can work through the list of Work Items and punch each off when complete. So, adding the capability for multiple Handle-It Work Items to show up would be helpful.

**Comment by rarafon** [comment]
In this scenario, do you mean being able to add multiple handleit work items from the vetting worksheet that would correspond to the same project or to be able to go to projects page to add additional work items to that handle-it project?

**Comment by dandahle** [comment]
I think the first option of adding 2 or more Handle-It Work Items in the Vetting Worksheet would be fine. (Practically speaking, once the Handle-It team is on site they will likely just fix whatever new shows up without updating, so adding in the Project view I don't think is needed.) But, being able to have a Handle-It Work Item to fix a faucet leak and a second one to replace a stair handle in the Vetting Worksheet would be useful.

**Comment by rarafon** [comment]
Being able to add in the project view would be a change of some variables so it is faster.

In the Vetting Worksheet, right now for non-handleit work items, it checks if there's an assessment and then adds any new work items to that. For the handle-it work items, would something like this work? 

![image](https://user-images.githubusercontent.com/11997170/112736064-17b4be00-8f0d-11eb-974c-7d16bbb6b4bf.png)

It was a quick html addition, so the spacing and whatnot is off, but when the user selects the handle-it option, there will be a select to allow the user to add the handleit work item to either a new handleit project or an existing one.

**Comment by dandahle** [comment]
I still see only one Handle-It for John Jones in the Handle-It Upcoming. There are two Completed Handle-Its and one Upcoming; however there are 4 total Handle-It work items in the application VW. The "Repair leak under sink" Handle-It work item only shows up in the Vetting Work Sheet and not the Projects tab. That should show up somewhere. Ideally we could group Handle-Its so a project reflects one visit to do all the Handle-Its in a project. 

Also, not all the Work Items in the John Jones application are accounted for in the Handle-It projects, Site Assessment and Projects. Only the last two Work Items show up in Assessment. The ones at the top of the list (Fix rotted siding, Repair roof, Porta potty, Waste removal and New all show To Review status)  are not Handle-It are not accounted for in Assessment or Project.

**Comment by dandahle** [comment]
I added a new application (Reese's Millmore) with no previous work item history to test this. I made two Handle-It Work Items in vetting and they show up separately in Projects. 

If you implemented your suggested fix, would the first Handle-It be "New" and then the second would have the option of being added to "Existing" and then choose the just made "New" Handle-It? 

One point of clarification, is that the Handle-Its would always be for the same client, so one option would be that if that Client has any Handle-It in Upcoming, the Handle-It Work Item would automatically be added to that open Handle-It project.

**Comment by JJMilmore** [comment]
Under Handle-It projects, I don't see anyway to add more work items.

**Comment by dandahle** [comment]
I just logged in with my dan@corvus credentials and added 3 new Handle-Its in the Vetting Worksheet view for JJ Smith. I didn't see any new methods for adding Handle-Its. I also restarted both MongoDB and the nodeJS and checked that the latest code had been pulled, which it had.  

Andy - Is this something you have an implementation for? 

If not, may I suggest a different solution? Is there a way to gather all the open Handle-Its (i.e. any Handle-It that does *not* have a Completed or Withdrawn status) for a given client into one Handle-It Report? That would address the issue we're trying to get at here to both consolidate the view of work to be done by a Handle-It team and make things simpler and straight forward for the client.
